api/handlers/clients/post_test.go

```diff
@@ -1 +1,54 @@
 package clients_test
+
+import (
+	"database/sql"
+	"net/http"
+	"testing"
+
+	"github.com/fanfit/login/models/clients/repository"
+	"github.com/fanfit/login/testutil"
+)
+
+var payload = "{\"ID\":0,\"UserTypeID\":0,\"FirstName\":\"Gabe\",\"LastName\":\"\",\"Email\":\"\",\"CreatedDate\":\"0001-01-01T00:00:00Z\",\"Username\":{\"String\":\"\",\"Valid\":false},\"PhoneNo\":{\"String\":\"\",\"Valid\":false},\"Gender\":{\"String\":\"\",\"Valid\":false},\"ProfilePicture\":{\"String\":\"\",\"Valid\":false},\"Bio\":{\"String\":\"\",\"Valid\":false},\"FanfitUserID\":0,\"TempField\":{\"String\":\"\",\"Valid\":false}}"
+
+var testUser2 = repository.GetClientByIDRow{
```

Comment:
Do we need two test users ?

Reply:
We don't even need post_test in the client service 
This is because the only time we actually make clients is in userservice

Reply:
I don't follow. Can you elaborate?